stdlib/Distributed/test/distributed_exec.jl

```diff
@@ -428,10 +428,10 @@ try
     end
     error("unexpected")
 catch ex
-    @test typeof(ex) == CompositeException
+    @test typeof(ex) === CompositeException
     @test length(ex) == 5
-    @test typeof(ex.exceptions[1]) == TaskFailedException
-    @test typeof(ex.exceptions[1].task.exception) == ErrorException
+    @test typeof(ex.exceptions[1]) === TaskFailedException
+    @test typeof(ex.exceptions[1].task.exception) === ErrorException
```

Comment:
I think these should be `isa` checks, since they theoretically could become abstract types without the API (and thus the test) being violated